bpfd/src/server/bpf.rs

```diff
@@ -253,6 +256,12 @@ impl<'a> BpfManager<'a> {
             (None, 0)
         };
 
+        let mut lcl_proceed_on = proceed_on;
```

Comment:
You _should_ just be able to do something like this to remove the lcl_proceed_on intermediate
```
let proceed_on = if proceed_on.is_empty() { 
    vec![DEFAULT_XDP_PROCEED_ON_PASS, DEFAULT_XDP_PROCEED_ON_DISPATCHER_RETURN]
} else {
    proceed_on
};
```

See playground example -> https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=6694380c8c83082c909c2dd4998966bc